packages/lib/src/acting-io.ts

```diff
@@ -42,9 +42,13 @@ export async function selectAndActOnGeneric<ID, O, L>(command: SelectingCommand<
 		chosenId = await getIdFromUser(command, list)
 	}
 	const updatedItem = await actionFunction(chosenId)
+
+	if (successMessage) {
+		process.stdout.write(`${successMessage.replace('{{id}}', JSON.stringify(chosenId))}\n`)
```

Comment:
Github won't let me comment on the code I think you should change since you haven't touched it but you could actually remove this line from the `selectAndActOn` method below and instead pass its `successMessage` parameter on to this function.